cfy_cluster_setup/utils.py

```diff
@@ -191,3 +203,64 @@ def yum_is_present():
 
 def current_host_ip():
     return socket.gethostbyname(socket.getfqdn())
+
+
+def _check_ssl_file(filename, kind='Key'):
+    """Does the cert/key file valid?"""
+    if kind == 'Key':
```

Comment:
I'd probably make these lower cases if I was using a fake-enum. Alternatively, as it only supports certs and keys you could just make it a bool- key=True?

Reply:
Alternatively, make this function take the paths for both?

Reply:
Thanks, separated it into two functions for convenience.